### PR TITLE
fix: Change log format to json

### PR DESCRIPTION
**Pull request by tcar121293**
Change log format to json
## Description
Change log format to json
## Related Issue Or Context

Closes: #67 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation

## Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have ensured that all acceptance criteria (or expected behavior) from issue are met
- [ ] I have updated the documentation locally and in chainbridge-docs.
- [ ] I have added tests to cover my changes.
- [ ] I have ensured that all the checks are passing and green, I've signed the CLA bot

### CODE REVIEW

File: src/utils/logger.ts [hunk 1]
```diff
@@ -1,68 +1,29 @@
...
-      datePattern: "YYYY-MM-DD",
-      dirname: logDir + "/error", // log file /logs/error/*.log in save
-      filename: "%DATE%.log",
-      maxFiles: 30, // 30 Days saved
-      handleExceptions: true,
-      json: false,
-      zippedArchive: true,
-    }),
-  ],
+const format = winston.format.printf(({ level, message, labels, timestamp, requestId }) => {
```
Comment (mpetrun5) [inline src/utils/logger.ts:4]: Do ne need this now anymore?

File: package.json [hunk 2]
```diff
@@ -38,7 +38,8 @@
     "node-cache": "^5.1.2",
     "node-cleanup": "^2.1.2",
     "winston": "^3.3.3",
-    "winston-daily-rotate-file": "^4.5.5"
+    "winston-daily-rotate-file": "^4.5.5",
```
Comment (mpetrun5) [inline package.json:41]: This is extra now.

**Review by mpetrun5 — CHANGES_REQUESTED** [review]

File: src/utils/logger.ts (hunk 1 above)
Comment (tcar121293) [reply src/utils/logger.ts:4]: With this, logs are better formated. Eg. With this code:
```indexer_1           | 2023-08-01T07:43:16.732Z [explorer-indexer] INFO: 	indexed block on evm2: 560, domainID: 2```

without:
```indexer_1           | {"labels":{"module":"explorer-indexer"},"level":"info","message":"indexed block on evm2: 570, domainID: 2"}```

Comment (mpetrun5) [reply src/utils/logger.ts:4]: But would this be indexed correctly on datadog then?

File: src/utils/logger.ts [hunk 3]
```diff
@@ -1,68 +1,29 @@
...
+  message = winston.format.colorize({ all: false, message: true }).colorize(level, message as string)
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
```
Comment (mpetrun5) [inline src/utils/logger.ts:6]: Might as well cast it instead of doing this.

**Review by mpetrun5 — APPROVED** [review]

**Pull request merged**